easybuild/easyconfigs/t/torchdata-0.3.0-PyTorch-1.11.0.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'PythonBundle'
+
+name = 'torchdata'
+version = '0.3.0'
+local_python_version = '3.9.5'
+local_pytorch_version = '1.11.0'
+versionsuffix = '-PyTorch-%s' % local_pytorch_version
+local_pytorch_versionsuffix = '-foss-2021a-CUDA-11.3.1'
+
+homepage = 'https://github.com/pytorch/data'
+description = "TorchData is a prototype library of common modular data loading primitives for easily constructing flexible and performant data pipelines."
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+dependencies = [
+    ('Python', local_python_version),
+    ('PyTorch', local_pytorch_version, local_pytorch_versionsuffix),
+]
+
+sanity_pip_check = True
+use_pip = True
+
+exts_list = [
+    (name, version, {
+        'source_tmpl': 'v%(version)s.tar.gz',
```

Comment:
This could come bite us with conflicting filenames if used as part of a bigger bundle

Hm, do we not have a way of specifying download-name in extensions?

Reply:
ok we do, e.g.
```
'sources': [{'download_filename': 'v%(version)s.tar.gz', 'filename': '%(name)s-%(version)s.tar.gz'}],
```

Reply:
```suggestion
        'sources': [{'download_filename': 'v%(version)s.tar.gz', 'filename': '%(name)s-%(version)s.tar.gz'}],
```
changing to

Reply:
Ignore this. revisit this idea if we decide to bundle this with PyTorch

Reply:
This doesn't matter for this single thing here. But you can do the download filename if you want to. Would make it easier to just copy this over to a bundle if we decide to.